asio/README.txt

```diff
@@ -4,10 +4,8 @@ In order to build ASIO support into Pd on Windows, you need to download the
 ASIO sources from Steinberg directly.  Their license does not let us
 redistribute their source files.
 
-- Download the ASIO SDK http://www.steinberg.net/en/company/3rd_party_developer.html
- You have to agree to Steinberg's license, then submit an email address, then
- they'll send you the download URL in an email. 
-
-- Uncompress asiosdk2.2.zip into pd/asio
+- Download the ASIO SDK https://www.steinberg.net/en/company/developers.html
+- Uncompress asiosdk2.3.zip into /asio
+- the ASIO source files should end up in /asio/ASIOSDK2.3
```

Comment:
Maybe the better suggestion is to use "/asio/ASIOSDK" and point the Makefile there to avoid having to change this again when the Asio 2.4 ASK comes out (if ever)...